hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/Text.java

```diff
@@ -73,6 +73,10 @@ protected CharsetDecoder initialValue() {
     }
   };
 
+  // max size of the byte array, seems to be a safe choice for multiple VMs
```

Comment:
Multiple VMs?

Reply:
Maybe I should have written different kind of VMs (OpenJDK, HotSpot, etc). It's more like a practical value that will likely work under different versions.

Reply:
JVMs then